packages/contracts/contracts/plugin/PluginRepo.sol

```diff
@@ -68,14 +73,33 @@ contract PluginRepo is
     /// @param invalidContract The address not being a contract.
     error InvalidContractAddress(address invalidContract);
 
-    /// @notice Emitted when a new version is created.
-    /// @param versionId The version index.
-    /// @param semanticVersion The semantic version number.
+    /// @notice Thrown if release id is 0.
+    error ReleaseNull();
+
+    /// @notice Thrown if release id is by more than 1 to the previous release id.
+    /// @param currentRelease the current latest release id.
+    /// @param newRelease new release id dev is trying to push.
+    error ReleaseTooBig(uint256 currentRelease, uint256 newRelease);
```

Comment:
Not a good error name.

How about `ReleaseIdIncrementInvalid`?

Reply:
https://github.com/aragon/core/pull/164/commits/c2e2a97d59f699a005149fdffa639a3467621ffe